introduction.md

```diff
@@ -1,2 +1,7 @@
 Hey! My name is Ricardo. I just dropped my phd course, so I'm back to the coding game!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?